src/UglyToad.PdfPig/Util/ColorSpaceDetailsParser.cs

```diff
@@ -48,7 +48,8 @@ internal static class ColorSpaceDetailsParser
             {
                 if (cannotRecurse)
                 {
-                    return UnsupportedColorSpaceDetails.Instance;
+                    // Not sure if always Gray, it's just a single color.
+                    return DeviceGrayColorSpaceDetails.Instance;

```

Comment:
`UnsupportedColorSpaceDetails` was refering to `DeviceGray` `enum` as color space. We don't use the `enum` anymore so we need the actual color space.